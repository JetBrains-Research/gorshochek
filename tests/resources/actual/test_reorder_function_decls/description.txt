transformation_1
		reorder function decls
transformation_2
		reorder function decls
