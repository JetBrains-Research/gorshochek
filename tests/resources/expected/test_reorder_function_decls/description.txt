transformation_1
		reorder function decls
